### PR TITLE
Fixed chokepoints

### PR DESCRIPTION
**Pull request by ArnauPrat**
Removed unneeded or duplicated chokepoints

### CODE REVIEW

**Comment by ArnauPrat** [comment]
Honeslty, I don't understand why so many "merge" commits in this pull request...

File: choke-points.tex [hunk 1]
```diff
@@ -265,8 +230,10 @@ \section{RDF and Graph Specifics}
 
 \cpSection{7.1}{QEXE}{Path pattern reuse}
 
-This choke-point tests the ability of the execution engine to reuse work across graph traversals \todo{complete in more detail}
-For example, when computing paths within a range of distances, it is often possible to incrementally compute longer paths by reusing paths of shorter distances that were already computed
+This choke-point tests the ability of the execution engine to reuse work across
+graph traversals. For example, when computing paths within a range of distances,
+it is often possible to incrementally compute longer paths by reusing paths of
+shorter distances that were already computed
```
Comment (szarnyasg) [inline choke-points.tex]: missing `.`

File: choke-points.tex [hunk 2]
```diff
@@ -206,7 +169,9 @@ \section{Expression Calculation}
 
 \cpSection{4.4}{QEXE}{String matching performance}
 
-\todo{this is not covered by the queries - at least according to the docs}
+This chokepoint tests the ability of efficiently evaluate complex string
```
Comment (szarnyasg) [inline choke-points.tex]: `evaluating`

File: choke-points.tex [hunk 3]
```diff
@@ -206,7 +169,9 @@ \section{Expression Calculation}
...
+matching expressions (e.g. via regular expressions).
```
Comment (szarnyasg) [inline choke-points.tex]: minor remark: use the `\eg` command for `e.g.`

**Pull request merged**